### PR TITLE
getP2P params poc

### PR DESCRIPTION
**Pull request by jmacmahonr3**

### CODE REVIEW

File: components/link-manager/src/main/kotlin/net/corda/p2p/linkmanager/common/MessageConverter.kt [hunk 1]
```diff
@@ -232,8 +232,8 @@ class MessageConverter {
                         "which is not in the network map. The message was discarded.")
                 return null
             }
-            val groupPolicy = groupPolicyProvider.getGroupPolicy(source)
-            if (groupPolicy == null) {
+            val p2pParams = groupPolicyProvider.getP2PParameters(source)
+            if (p2pParams == null) {
                 logger.warn(
                     "Could not find the group info in the " +
```
Comment (yarivus) [inline components/link-manager/src/main/kotlin/net/corda/p2p/linkmanager/common/MessageConverter.kt:238]: msg should be changed as you're checking for something else

File: components/link-manager/src/main/kotlin/net/corda/p2p/linkmanager/sessions/SessionManagerImpl.kt [hunk 2]
```diff
@@ -432,8 +426,8 @@ internal class SessionManagerImpl(
         multiplicity: Int
     ): List<Pair<AuthenticationProtocolInitiator, InitiatorHelloMessage>> {
 
-        val groupPolicy = groupPolicyProvider.getGroupPolicy(counterparties.ourId)
-        if (groupPolicy == null) {
+        val p2pParams = groupPolicyProvider.getP2PParameters(counterparties.ourId)
+        if (p2pParams == null) {
             logger.warn(
                 "Could not find the group information in the GroupPolicyProvider for ${counterparties.ourId}." +
```
Comment (yarivus) [inline components/link-manager/src/main/kotlin/net/corda/p2p/linkmanager/sessions/SessionManagerImpl.kt:432]: msg should be changed

File: components/link-manager/src/main/kotlin/net/corda/p2p/linkmanager/sessions/SessionManagerImpl.kt [hunk 3]
```diff
@@ -549,8 +544,8 @@ internal class SessionManagerImpl(
             return null
         }
 
-        val groupPolicy = groupPolicyProvider.getGroupPolicy(sessionCounterparties.ourId)
-        if (groupPolicy == null) {
+        val p2pParams = groupPolicyProvider.getP2PParameters(sessionCounterparties.ourId)
+        if (p2pParams == null) {
             logger.warn(
                 "Could not find the group information in the GroupPolicyProvider for ${sessionCounterparties.ourId}." +
```
Comment (yarivus) [inline components/link-manager/src/main/kotlin/net/corda/p2p/linkmanager/sessions/SessionManagerImpl.kt:550]: msg should be changed

File: components/link-manager/src/main/kotlin/net/corda/p2p/linkmanager/sessions/SessionManagerImpl.kt [hunk 4]
```diff
@@ -762,8 +757,8 @@ internal class SessionManagerImpl(
         }
 
         val (hostedIdentityInSameGroup, peerMemberInfo) = locallyHostedIdentityWithPeerMemberInfo
-        val groupPolicy = groupPolicyProvider.getGroupPolicy(hostedIdentityInSameGroup)
-        if (groupPolicy == null) {
+        val p2pParams = groupPolicyProvider.getP2PParameters(hostedIdentityInSameGroup)
+        if (p2pParams == null) {
             logger.couldNotFindGroupInfo(message::class.java.simpleName, message.header.sessionId, hostedIdentityInSameGroup)
```
Comment (yarivus) [inline components/link-manager/src/main/kotlin/net/corda/p2p/linkmanager/sessions/SessionManagerImpl.kt:762]: different logger func to call ?!

**Review by yarivus — COMMENTED** [review]
some minor comments

**Comment by jmacmahonr3** [comment]
Hey @yarivus 
This is a poc pr likely to be closed within the week as @SaurabhR3 will be raising a replacement one that extends the concepts here.

**Pull request closed without merging**